domains/runtime/evm/src/fees.rs

```diff
@@ -0,0 +1,98 @@
+use crate::{AccountId, Balance, Balances, OperatorRewards, Runtime, RuntimeCall};
+use codec::Encode;
+use frame_support::traits::{Currency, ExistenceRequirement, Get, Imbalance, WithdrawReasons};
+use pallet_balances::NegativeImbalance;
+use sp_runtime::traits::{DispatchInfoOf, PostDispatchInfoOf, Zero};
+use sp_runtime::transaction_validity::{InvalidTransaction, TransactionValidityError};
+
+pub struct DomainTransactionByteFee;
```

Comment:
We should introduce this in `pallet-operator-rewards` instead and reuse it across prod and test runtimes instead